grunt/config/copy.js

```diff
@@ -1,240 +1,207 @@
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>components/**/assets/**'],
-                    dest: '<%= outputdir %>assets/',
-                    filter: function(filepath) {
-                        return grunt.config('helpers').includedFilter(filepath);
-                    },
-                    rename: _.partial(collate, "assets")
-                }
-            ]
+        rename: _.partial(collate, "fonts")
+      }]
+    },
+    extensionAssets: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>extensions/**/assets/**'],
+        dest: '<%= outputdir %>assets/',
+        filter: function(filepath) {
+          return grunt.config('helpers').includedFilter(filepath);
         },
-        componentFonts: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>components/**/fonts/**'],
-                    dest: '<%= outputdir %>fonts/',
-                    filter: function(filepath) {
-                        return grunt.config('helpers').includedFilter(filepath);
-                    },
-                    rename: _.partial(collate, "fonts")
-                }
-            ]
+        rename: _.partial(collate, "assets")
+      }]
+    },
+    extensionFonts: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>extensions/**/fonts/**'],
+        dest: '<%= outputdir %>fonts/',
+        filter: function(filepath) {
+          return grunt.config('helpers').includedFilter(filepath);
         },
-        extensionAssets: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>extensions/**/assets/**'],
-                    dest: '<%= outputdir %>assets/',
-                    filter: function(filepath) {
-                        return grunt.config('helpers').includedFilter(filepath);
-                    },
-                    rename: _.partial(collate, "assets")
-                }
-            ]
+        rename: _.partial(collate, "fonts")
+      }]
+    },
+    menuAssets: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>menu/<%= menu %>/assets/**'],
+        dest: '<%= outputdir %>assets/',
+        filter: function(filepath) {
+          return grunt.config('helpers').includedFilter(filepath);
         },
-        extensionFonts: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>extensions/**/fonts/**'],
-                    dest: '<%= outputdir %>fonts/',
-                    filter: function(filepath) {
-                        return grunt.config('helpers').includedFilter(filepath);
-                    },
-                    rename: _.partial(collate, "fonts")
-                }
-            ]
+        rename: _.partial(collate, "assets")
+      }]
+    },
+    coreFonts: {
+      files: [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?